src/components/OAuthLoginButton.tsx

```diff
@@ -24,12 +36,62 @@ export const OAuthLoginButton: FC<OAuthLoginButtonParams> = ({
   }, [login, onSuccess, onFail]);
 
   return (
-    <TouchableOpacity
-      testID={tID('oauth-login-button')}
-      {...touchableOpacityProps}
-      onPress={_login}
-    >
-      {children}
-    </TouchableOpacity>
+    <View style={styles.button}>
+      <TouchableOpacity
+        testID={tID('oauth-login-button')}
+        {...touchableOpacityProps}
+        onPress={_login}
+        style={styles.touchableOpacity}
+      >
+        <View style={styles.content}>
+          <Text style={styles.label}>{label}</Text>
+        </View>
+      </TouchableOpacity>
+    </View>
   );
 };
+
+const defaultStyles = (theme: Theme) => {
+  const button: ViewStyle = {
+    flexDirection: 'row',
+  };
+  const touchableOpacity: ViewStyle = {
+    alignItems: 'center',
+    backgroundColor: theme.colors.secondary,
+    borderRadius: 5 * theme.roundness,
+
+    minWidth: 64,
+  };
+
+  const content: ViewStyle = {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+  };
+
+  const label: TextStyle = {
+    color: theme.colors.onPrimary,
+    marginVertical: 10,
+    marginHorizontal: 24,
+
+    fontFamily: Platform.select({
+      web: 'Roboto, "Helvetica Neue", Helvetica, Arial, sans-serif',
+      ios: 'System',
+      default: 'sans-serif-medium',
+    }),
+
+    fontWeight: '500',
+    letterSpacing: 0.1,
+    lineHeight: 20,
+    fontSize: 14,
+  };
```

Comment:
Lots of this will be boilerplate for many components and should be moved to the theme.  A good example of how this can be done is found here:

https://github.com/callstack/react-native-paper/blob/main/src/styles/themes/v3/tokens.tsx

Reply:
👍 Great offline discussion on RN paper too.  I'm all for considering adopting that to avoid having to write too much code for things like Button, Text, icons, what props should a theme have, etc.  Maybe their providers could be wrapped by ours if/where needed.  Let's maybe shoot for a 1 day prototype on this next week